quickstart.mdx

```diff
@@ -31,7 +31,37 @@ You can find your URL from the Overview page of the dashboard.
 
 This URL is instantly available and will update whenever you make changes to your documentation. It's perfect for testing and sharing with your team during development.
 
-## Development Workflows
+### Install the GitHub App
+
+Mintlify provides a GitHub App that automates the deployment process when you push changes to your repository.
+
+You can install the GitHub App by following the instructions from the onboarding checklist or from your dashboard.
+
+1. Navigate to **Settings** in your Mintlify dashboard.
+2. Select **GitHub App** from the sidebar.
+3. Select **Install GitHub App**. This will open a new tab to the GitHub App installation page.
+4. Select the organization or user account where you want to install the app. Then select the repositories that you want t oconnect.
+
+<Frame>
+  <img src="/images/quickstart/github-app-installation-light.png" alt="GitHub App Installation" className="block dark:hidden" />
+  <img src="/images/quickstart/github-app-installation-dark.png" alt="GitHub App Installation" className="hidden dark:block" />
+</Frame>
+
+<Info>
+  Remember to update the GitHub App permissions if you move the documentation to a different repository.
+</Info>
+
+### Authorize your GitHub Account
```

Comment:
just to note - authorizing is not required. it is purely for associating commits with the proper author if you're making edits through the web editor.

Reply:
it might be worth taking it out of the quickstart and putting it elsewhere